app/views/contact_methods/_call_us.html.erb

```diff
@@ -1,11 +1,9 @@
-<div class="l-contact-panel">
-  <div class="contact-panel contact-panel--last">
-    <h2 aria-level="2" class="contact-panel__heading t-contact-heading" role="heading"><span class="icon icon--call-us"></span><%= t('contact_panels.call_us.title') %></h2>
-    <p class="t-contact-introduction"><%= t('contact_panels.call_us.description') %></p>
-    <p><a class="contact-panel__telephone t-contact-number" href="<%= t('contact_panels.call_us.telephone_number_url') %>"><%= t('contact_panels.call_us.telephone_number') %> *</a></p>
+<div class="contact-panel contact-panel--last">
+  <h2 aria-level="2" class="contact-panel__heading t-contact-heading" role="heading"><span class="icon icon--call-us"></span><%= t('contact_panels.call_us.title') %></h2>
+  <p class="t-contact-introduction"><%= t('contact_panels.call_us.description') %></p>
+  <p><a class="contact-panel__telephone t-contact-number" href="<%= t('contact_panels.call_us.telephone_number_url') %>"><%= t('contact_panels.call_us.telephone_number') %> *</a></p>
```

Comment:
I think it would be better to put the `.contact-panel__telephone` class on the containing `<p>` tag here and use it to take out the default vertical margins. In general the vertical spacing on this panel looks a little wide to me and inconsistent with other panels.

Reply:
This class only affects the typography and set some margins in larger screens which I think are the same for others panels, or at least they look the same